code/modules/tetris/_tetris_defines.dm

```diff
@@ -0,0 +1,15 @@
+#define TETRIS_LOCK_INFINITY		1				//can stall forever by turning
+#define TETRIS_LOCK_LIMITED			2				//can turn <x> number of times to stall
+#define TETRIS_LOCK_STEP			3				//can only delay by making piece move down naturally
+
+#define TETRIS_FIELD_WIDTH					10
+#define TETRIS_FIELD_HEIGHT					40
+#define TETRIS_FIELD_HEIGHT_PLACEMENT		22
+#define TETRIS_FIELD_HEIGHT_SHOW			21
+
+#define TETRIS_PIECE_BAG_DEPTH			7
```

Comment:
Guideline says these should be called "tetriminos", not "pieces".

Reply:
*tetrominoes

Reply:
Damnit.